components/buttons/buttons.css

```diff
@@ -0,0 +1,49 @@
+.btn{
+    width: max-content; 
+    letter-spacing: 1px;
+    background-color: #1FABD7;
+    padding: 0.8rem 0.8rem;
+    font-size: 0.8rem;
+    position: relative;
+    overflow: hidden;
+    border-radius: 1rem;
+    font-weight: bold;
+    border: transparent;
+}
+
+.btn-primary:hover{
+    background-color: #1a9fc7;
+}
+
+.btn-primary{
+    color: whitesmoke;
+}
+
+.btn-secondary{
+    color: #1FABD7;
```

Comment:
colors can go to a variables

Reply:
Hi siddhant , I have not created base file yet. Once the components are done will replace the colour with variable. Will this approach work.